[CI] DocsClientYamlTestSuiteIT test {yaml=reference/docs/termvectors/line_171} failure on 7.6

Build log: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.6+multijob-unix-compatibility/os=oraclelinux-7&&immutable/84/console

Build scan: https://gradle-enterprise.elastic.co/s/agilnuf73b2e6

Failure:
```
Failure at [reference/docs/termvectors:177]: $body didn't match expected value:
                         $body: 
                             _id: same [1]
                          _index: same [twitter]
                           _type: same [_doc]
                        _version: same [1]
                           found: same [true]
                    term_vectors: 
                              text: 
                    field_statistics: 
                             doc_count: expected Integer [2] but was Integer [1]
                          sum_doc_freq: expected Integer [6] but was Integer [2]
                               sum_ttf: expected Integer [8] but was Integer [4]
                               terms: 
                                  test: 
                                doc_freq: expected Integer [2] but was Integer [1]
                               term_freq: same [3]
                                  tokens: 
                                         0: 
                                  end_offset: same [12]
                                     payload: same [d29yZA==]
                                    position: same [1]
                                start_offset: same [8]
                                         1: 
                                  end_offset: same [17]
                                     payload: same [d29yZA==]
                                    position: same [2]
                                start_offset: same [13]
                                         2: 
                                  end_offset: same [22]
                                     payload: same [d29yZA==]
                                    position: same [3]
                                start_offset: same [18]
                                     ttf: expected Integer [4] but was Integer [3]
                               twitter: 
                                doc_freq: expected Integer [2] but was Integer [1]
                               term_freq: same [1]
                                  tokens: 
                                         0: 
                                  end_offset: same [7]
                                     payload: same [d29yZA==]
                                    position: same [0]
                                start_offset: same [0]
                                     ttf: expected Integer [2] but was Integer [1]
                            took: same [0]
java.lang.AssertionError: 
Failure at [reference/docs/termvectors:177]: $body didn't match expected value:
                         $body: 
                             _id: same [1]
                          _index: same [twitter]
                           _type: same [_doc]
                        _version: same [1]
                           found: same [true]
                    term_vectors: 
                              text: 
                    field_statistics: 
                             doc_count: expected Integer [2] but was Integer [1]
                          sum_doc_freq: expected Integer [6] but was Integer [2]
                               sum_ttf: expected Integer [8] but was Integer [4]
                               terms: 
                                  test: 
                                doc_freq: expected Integer [2] but was Integer [1]
                               term_freq: same [3]
                                  tokens: 
                                         0: 
                                  end_offset: same [12]
                                     payload: same [d29yZA==]
                                    position: same [1]
                                start_offset: same [8]
                                         1: 
                                  end_offset: same [17]
                                     payload: same [d29yZA==]
                                    position: same [2]
                                start_offset: same [13]
                                         2: 
                                  end_offset: same [22]
                                     payload: same [d29yZA==]
                                    position: same [3]
                                start_offset: same [18]
                                     ttf: expected Integer [4] but was Integer [3]
                               twitter: 
                                doc_freq: expected Integer [2] but was Integer [1]
                               term_freq: same [1]
                                  tokens: 
                                         0: 
                                  end_offset: same [7]
                                     payload: same [d29yZA==]
                                    position: same [0]
                                start_offset: same [0]
                                     ttf: expected Integer [2] but was Integer [1]
                            took: same [0]

Caused by: java.lang.AssertionError: 
$body didn't match expected value:
                         $body: 
                             _id: same [1]
                          _index: same [twitter]
                           _type: same [_doc]
                        _version: same [1]
                           found: same [true]
                    term_vectors: 
                              text: 
                    field_statistics: 
                             doc_count: expected Integer [2] but was Integer [1]
                          sum_doc_freq: expected Integer [6] but was Integer [2]
                               sum_ttf: expected Integer [8] but was Integer [4]
                               terms: 
                                  test: 
                                doc_freq: expected Integer [2] but was Integer [1]
                               term_freq: same [3]
                                  tokens: 
                                         0: 
                                  end_offset: same [12]
                                     payload: same [d29yZA==]
                                    position: same [1]
                                start_offset: same [8]
                                         1: 
                                  end_offset: same [17]
                                     payload: same [d29yZA==]
                                    position: same [2]
                                start_offset: same [13]
                                         2: 
                                  end_offset: same [22]
                                     payload: same [d29yZA==]
                                    position: same [3]
                                start_offset: same [18]
                                     ttf: expected Integer [4] but was Integer [3]
                               twitter: 
                                doc_freq: expected Integer [2] but was Integer [1]
                               term_freq: same [1]
                                  tokens: 
                                         0: 
                                  end_offset: same [7]
                                     payload: same [d29yZA==]
                                    position: same [0]
                                start_offset: same [0]
                                     ttf: expected Integer [2] but was Integer [1]
                            took: same [0]

[2020-02-26T05:37:31,016][INFO ][o.e.s.DocsClientYamlTestSuiteIT] [test] [yaml=reference/docs/termvectors/line_171] before test
[2020-02-26T05:37:32,480][INFO ][o.e.s.DocsClientYamlTestSuiteIT] [test] Stash dump on test failure [{
  "stash" : {
    "body" : {
      "_index" : "twitter",
      "_type" : "_doc",
      "_id" : "1",
      "_version" : 1,
      "found" : true,
      "took" : 0,
      "term_vectors" : {
        "text" : {
          "field_statistics" : {
            "sum_doc_freq" : 2,
            "doc_count" : 1,
            "sum_ttf" : 4
          },
          "terms" : {
            "test" : {
              "doc_freq" : 1,
              "ttf" : 3,
              "term_freq" : 3,
              "tokens" : [
                {
                  "position" : 1,
                  "start_offset" : 8,
                  "end_offset" : 12,
                  "payload" : "d29yZA=="
                },
                {
                  "position" : 2,
                  "start_offset" : 13,
                  "end_offset" : 17,
                  "payload" : "d29yZA=="
                },
                {
                  "position" : 3,
                  "start_offset" : 18,
                  "end_offset" : 22,
                  "payload" : "d29yZA=="
                }
              ]
            },
            "twitter" : {
              "doc_freq" : 1,
              "ttf" : 1,
              "term_freq" : 1,
              "tokens" : [
                {
                  "position" : 0,
                  "start_offset" : 0,
                  "end_offset" : 7,
                  "payload" : "d29yZA=="
                }
              ]
            }
          }
        }
      }
    }
  }
}]
```

Reproduce with:
```
./gradlew ':docs:integTestRunner' --tests "org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT.test {yaml=reference/docs/termvectors/line_171}" -Dtests.seed=47DA4629C197BF0
```

Does not reproduce locally for me.

--- a/docs/reference/docs/termvectors.asciidoc
+++ b/docs/reference/docs/termvectors.asciidoc
@@ -216,7 +216,7 @@ PUT /twitter/_doc/1
   "text" : "twitter test test test "
 }
 
-PUT /twitter/_doc/2
+PUT /twitter/_doc/2?refresh=wait_for
 {
   "fullname" : "Jane Doe",
   "text" : "Another twitter test ..."

